salt/iiif/config/etc-nginx-sites-enabled-loris.conf

```diff
@@ -10,14 +10,9 @@ map $http_host $robots_disallow {
 }
 
 server {
-    {% if salt['elife.only_on_aws']() %}
-        {% if salt['elife.cfg']('project.elb') %}
-        listen 80;
-        {% else %}
-        listen 443 ssl;
-        {% endif %}
-    {% else %}
     listen 80;
+    {% if salt['elife.only_on_aws']() %}
```

Comment:
in the end it turned out it's not really possible to run this without an ELB, as cache cleaning will break havoc on a single server. So we could drop port 443 at all.

Reply:
443 is needed because of HSTS. I don't need to actually use the server, just test that it builds. I opened another PR in builder that makes the default configuration load balanced and a standalone adhoc server the exception: https://github.com/elifesciences/builder/pull/187